src/main/java/com/khalsa_ji/ems/Designation.java

```diff
@@ -0,0 +1,104 @@
+//  Waheguru Ji!
+
+//  NOTE:
+//  1. jobID in Employee is same as designationID in Designation
+//  2. jobTitle in Employee is same as designation in Designation
+
+package com.khalsa_ji.ems;
```

Comment:
Originally, I thought of having an imaginary domain as <my-github-user-id>.com(i.e. _khalsa-ji.com_) instead of the default _example.com_. The issue with that was as per the Oracle Java docs, all the special characters in the domain names should be replaced with underscores for naming any package. That's why it resulted into _khalsa_ji.com_.

References:
- [https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html](https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html)
- [https://stackoverflow.com/questions/3179216/what-is-the-convention-for-word-separator-in-java-package-names](https://stackoverflow.com/questions/3179216/what-is-the-convention-for-word-separator-in-java-package-names)

If I am not wrong, I think you mean to say that it is good to avoid underscores in packages name as it looks somewhat odd and instead it is good to use some name that is crisp and short while remaining meaningful. Is it so?